pallets/living-assets-ownership/src/mock.rs

```diff
@@ -29,13 +40,13 @@ impl frame_system::Config for Test {
 	type Hash = H256;
 	type Nonce = Nonce;
 	type Hashing = BlakeTwo256;
-	type AccountId = u64;
+	type AccountId = H160;
```

Comment:
does it mean that H160 is a **must** now ? Cause the current blockchain does not use it.

Reply:
I don't know how to solve otherwise, any advice?